docs/sources/installation/debian.md

```diff
@@ -24,26 +24,40 @@ You can install Grafana using our official APT repository, by downloading a `.de
 
 If you install from the APT repository, then Grafana is automatically updated every time you run `apt-get update`. 
 
-The APT repository installs the open source edition of Grafana.
+| Grafana Version | Package | Repository |
+|-----------------|---------|------------|
+| Grafana OSS     | grafana | `https://packages.grafana.com/oss/deb stable main` |
+| Grafana OSS (Beta)     | grafana | `https://packages.grafana.com/oss/deb beta main` |
+| Grafana Enterprise     | grafana-enterprise | `https://packages.grafana.com/enterprise/deb stable main` |
+| Grafana Enterprise (Beta)     | grafana-enterprise | `https://packages.grafana.com/enterprise/deb beta main` |
 
-#### To install the latest stable release:
+> We recommend all users to install the Enterprise Edition of Grafana, which can be seamlessly upgraded with a Grafana Enterprise [subscription](https://grafana.com/products/enterprise/?utm_source=grafana-install-page).
+
+#### To install the latest Enterprise release:
```

Comment:
```suggestion
#### To install the latest Enterprise Edition:
```

@torkelo what do you prefer here? I'm partial to Enterprise release but I don't really have a strong opinion.

Reply:
To install the latest Enterprise edition: 

(lowercase e)